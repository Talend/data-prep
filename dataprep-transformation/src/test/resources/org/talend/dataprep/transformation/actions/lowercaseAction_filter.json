{
  "actions": [
    {
      "action": "lowercase",
      "parameters": {
        "column_id": "firstname",
        "scope": "column",
        "filter": {
          "eq": {
            "field": "nbCommands",
            "value": "13"
          }
        }
      }
    }
  ]
}