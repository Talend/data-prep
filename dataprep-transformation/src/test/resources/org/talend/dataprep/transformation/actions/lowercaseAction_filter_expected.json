{
  "records": [
    {
      "avgAmount": "668.61",
      "birth": "02-07-1964",
      "city": "Santa Fe",
      "firstname": "Benjamin",
      "id": "1",
      "lastname": "Quincy",
      "nbCommands": "72",
      "registration": "30-11-2007",
      "state": "NC",
      "tdpId": 1
    },
    {
      "avgAmount": "200.56",
      "birth": "01-02-1986",
      "city": "Columbia",
      "firstname": "calvin",
      "id": "2",
      "lastname": "Jefferson",
      "nbCommands": "13",
      "registration": "25-08-2008",
      "state": "TN",
      "tdpId": 2
    },
    {
      "avgAmount": "645.39",
      "birth": "05-05-1982",
      "city": "Montpelier",
      "firstname": "Ulysses",
      "id": "3",
      "lastname": "Johnson",
      "nbCommands": "68",
      "registration": "26-07-2008",
      "state": "AL",
      "tdpId": 3
    },
    {
      "avgAmount": "387.21",
      "birth": "18-09-1948",
      "city": "Salem",
      "firstname": "Andrew",
      "id": "4",
      "lastname": "Taft",
      "nbCommands": "54",
      "registration": "15-03-2008",
      "state": "NV",
      "tdpId": 4
    },
    {
      "avgAmount": "324.46",
      "birth": "27-08-1969",
      "city": "Little Rock",
      "firstname": "richard",
      "id": "5",
      "lastname": "Eisenhower",
      "nbCommands": "13",
      "registration": "10-09-2008",
      "state": "HI",
      "tdpId": 5
    },
    {
      "avgAmount": "537.4",
      "birth": "25-07-1980",
      "city": "Salem",
      "firstname": "Franklin",
      "id": "6",
      "lastname": "Taylor",
      "nbCommands": "85",
      "registration": "31-03-2007",
      "state": "IN",
      "tdpId": 6
    },
    {
      "avgAmount": "583.08",
      "birth": "02-12-1965",
      "city": "Trenton",
      "firstname": "Dwight",
      "id": "7",
      "lastname": "Tyler",
      "nbCommands": "97",
      "registration": "06-11-2007",
      "state": "AZ",
      "tdpId": 7
    },
    {
      "avgAmount": "969.24",
      "birth": "08-04-1976",
      "city": "Montpelier",
      "firstname": "Lyndon",
      "id": "8",
      "lastname": "Harrison",
      "nbCommands": "50",
      "registration": "20-11-2007",
      "state": "ME",
      "tdpId": 8
    },
    {
      "avgAmount": "327.41",
      "birth": "29-05-1954",
      "city": "Atlanta",
      "firstname": "George",
      "id": "9",
      "lastname": "Monroe",
      "nbCommands": "22",
      "registration": "30-08-2007",
      "state": "MN",
      "tdpId": 9
    },
    {
      "avgAmount": "753.16",
      "birth": "15-10-1955",
      "city": "Topeka",
      "firstname": "Woodrow",
      "id": "10",
      "lastname": "Washington",
      "nbCommands": "58",
      "registration": "10-12-2007",
      "state": "MO",
      "tdpId": 10
    },
    {
      "avgAmount": "556.56",
      "birth": "15-02-1962",
      "city": "Jefferson City",
      "firstname": "Millard",
      "id": "11",
      "lastname": "Grant",
      "nbCommands": "21",
      "registration": "18-06-2007",
      "state": "WA",
      "tdpId": 11
    },
    {
      "avgAmount": "236.24",
      "birth": "12-01-1979",
      "city": "Salem",
      "firstname": "Calvin",
      "id": "12",
      "lastname": "Tyler",
      "nbCommands": "53",
      "registration": "26-08-2008",
      "state": "MD",
      "tdpId": 12
    },
    {
      "avgAmount": "125.16",
      "birth": "10-09-1974",
      "city": "Dover",
      "firstname": "Martin",
      "id": "13",
      "lastname": "Polk",
      "nbCommands": "51",
      "registration": "05-06-2008",
      "state": "IL",
      "tdpId": 13
    },
    {
      "avgAmount": "513.93",
      "birth": "26-09-1956",
      "city": "Helena",
      "firstname": "Martin",
      "id": "14",
      "lastname": "Cleveland",
      "nbCommands": "40",
      "registration": "19-11-2008",
      "state": "ND",
      "tdpId": 14
    },
    {
      "avgAmount": "250.01",
      "birth": "06-02-1987",
      "city": "Harrisburg",
      "firstname": "Richard",
      "id": "15",
      "lastname": "Clinton",
      "nbCommands": "17",
      "registration": "06-04-2008",
      "state": "AK",
      "tdpId": 15
    },
    {
      "avgAmount": "913.87",
      "birth": "09-06-1954",
      "city": "Honolulu",
      "firstname": "Rutherford",
      "id": "16",
      "lastname": "Adams",
      "nbCommands": "94",
      "registration": "02-05-2008",
      "state": "WI",
      "tdpId": 16
    },
    {
      "avgAmount": "737.34",
      "birth": "21-10-1955",
      "city": "Charleston",
      "firstname": "Franklin",
      "id": "17",
      "lastname": "Wilson",
      "nbCommands": "56",
      "registration": "01-07-2008",
      "state": "IA",
      "tdpId": 17
    },
    {
      "avgAmount": "125.21",
      "birth": "27-07-1949",
      "city": "Indianapolis",
      "firstname": "Rutherford",
      "id": "18",
      "lastname": "Hayes",
      "nbCommands": "49",
      "registration": "21-04-2008",
      "state": "VT",
      "tdpId": 18
    },
    {
      "avgAmount": "444.58",
      "birth": "23-06-1987",
      "city": "Lincoln",
      "firstname": "john",
      "id": "19",
      "lastname": "Roosevelt",
      "nbCommands": "13",
      "registration": "16-11-2007",
      "state": "IN",
      "tdpId": 19
    },
    {
      "avgAmount": "688.83",
      "birth": "02-04-1972",
      "city": "Helena",
      "firstname": "Ulysses",
      "id": "20",
      "lastname": "Harrison",
      "nbCommands": "21",
      "registration": "24-04-2008",
      "state": "GA",
      "tdpId": 20
    },
    {
      "avgAmount": "447.09",
      "birth": "03-01-1953",
      "city": "Boise",
      "firstname": "Ronald",
      "id": "21",
      "lastname": "Hoover",
      "nbCommands": "65",
      "registration": "19-03-2008",
      "state": "RI",
      "tdpId": 21
    },
    {
      "avgAmount": "594.67",
      "birth": "26-09-1980",
      "city": "Jackson",
      "firstname": "Bill",
      "id": "22",
      "lastname": "Eisenhower",
      "nbCommands": "46",
      "registration": "16-03-2008",
      "state": "NJ",
      "tdpId": 22
    },
    {
      "avgAmount": "459.53",
      "birth": "10-12-1975",
      "city": "Austin",
      "firstname": "Chester",
      "id": "23",
      "lastname": "Tyler",
      "nbCommands": "74",
      "registration": "13-12-2007",
      "state": "DE",
      "tdpId": 23
    },
    {
      "avgAmount": "303.34",
      "birth": "16-01-1959",
      "city": "Montpelier",
      "firstname": "Herbert",
      "id": "24",
      "lastname": "Roosevelt",
      "nbCommands": "70",
      "registration": "04-11-2008",
      "state": "SC",
      "tdpId": 24
    },
    {
      "avgAmount": "410.09",
      "birth": "29-07-1954",
      "city": "Springfield",
      "firstname": "William",
      "id": "25",
      "lastname": "Arthur",
      "nbCommands": "38",
      "registration": "30-11-2007",
      "state": "WV",
      "tdpId": 25
    },
    {
      "avgAmount": "827.53",
      "birth": "29-12-1964",
      "city": "Salt Lake City",
      "firstname": "Richard",
      "id": "26",
      "lastname": "McKinley",
      "nbCommands": "48",
      "registration": "19-12-2008",
      "state": "NH",
      "tdpId": 26
    },
    {
      "avgAmount": "417.83",
      "birth": "23-09-1954",
      "city": "Charleston",
      "firstname": "Woodrow",
      "id": "27",
      "lastname": "Ford",
      "nbCommands": "40",
      "registration": "11-03-2008",
      "state": "WV",
      "tdpId": 27
    },
    {
      "avgAmount": "183.15",
      "birth": "27-10-1966",
      "city": "Raleigh",
      "firstname": "Harry",
      "id": "28",
      "lastname": "Clinton",
      "nbCommands": "14",
      "registration": "17-10-2007",
      "state": "LA",
      "tdpId": 28
    },
    {
      "avgAmount": "645.45",
      "birth": "14-06-1948",
      "city": "Columbus",
      "firstname": "Richard",
      "id": "29",
      "lastname": "Nixon",
      "nbCommands": "6",
      "registration": "28-03-2007",
      "state": "NV",
      "tdpId": 29
    },
    {
      "avgAmount": "201.83",
      "birth": "18-01-1982",
      "city": "Des Moines",
      "firstname": "Thomas",
      "id": "30",
      "lastname": "Roosevelt",
      "nbCommands": "75",
      "registration": "18-12-2008",
      "state": "AL",
      "tdpId": 30
    },
    {
      "avgAmount": "605.52",
      "birth": "23-10-1955",
      "city": "Jackson",
      "firstname": "Bill",
      "id": "31",
      "lastname": "Harrison",
      "nbCommands": "31",
      "registration": "26-09-2008",
      "state": "VA",
      "tdpId": 31
    },
    {
      "avgAmount": "643.28",
      "birth": "02-04-1979",
      "city": "Cheyenne",
      "firstname": "George",
      "id": "32",
      "lastname": "Taft",
      "nbCommands": "48",
      "registration": "23-09-2007",
      "state": "OR",
      "tdpId": 32
    },
    {
      "avgAmount": "851.88",
      "birth": "09-01-1972",
      "city": "Nashville",
      "firstname": "Ronald",
      "id": "33",
      "lastname": "Lincoln",
      "nbCommands": "83",
      "registration": "08-06-2008",
      "state": "PA",
      "tdpId": 33
    },
    {
      "avgAmount": "354.77",
      "birth": "21-09-1980",
      "city": "Springfield",
      "firstname": "Woodrow",
      "id": "34",
      "lastname": "Pierce",
      "nbCommands": "15",
      "registration": "21-10-2007",
      "state": "FL",
      "tdpId": 34
    },
    {
      "avgAmount": "617.82",
      "birth": "11-09-1977",
      "city": "Richmond",
      "firstname": "Ronald",
      "id": "35",
      "lastname": "Reagan",
      "nbCommands": "4",
      "registration": "01-12-2008",
      "state": "MD",
      "tdpId": 35
    },
    {
      "avgAmount": "875.45",
      "birth": "08-01-1984",
      "city": "Sacramento",
      "firstname": "Herbert",
      "id": "36",
      "lastname": "Grant",
      "nbCommands": "26",
      "registration": "13-08-2007",
      "state": "TX",
      "tdpId": 36
    },
    {
      "avgAmount": "518.38",
      "birth": "03-10-1983",
      "city": "Raleigh",
      "firstname": "Benjamin",
      "id": "37",
      "lastname": "Fillmore",
      "nbCommands": "46",
      "registration": "16-03-2007",
      "state": "NH",
      "tdpId": 37
    },
    {
      "avgAmount": "109.2",
      "birth": "19-10-1973",
      "city": "Sacramento",
      "firstname": "Abraham",
      "id": "38",
      "lastname": "Fillmore",
      "nbCommands": "86",
      "registration": "15-05-2008",
      "state": "KY",
      "tdpId": 38
    },
    {
      "avgAmount": "245.09",
      "birth": "04-08-1986",
      "city": "Dover",
      "firstname": "Ulysses",
      "id": "39",
      "lastname": "Cleveland",
      "nbCommands": "48",
      "registration": "18-11-2007",
      "state": "OR",
      "tdpId": 39
    },
    {
      "avgAmount": "347.75",
      "birth": "13-11-1971",
      "city": "Denver",
      "firstname": "Andrew",
      "id": "40",
      "lastname": "Reagan",
      "nbCommands": "60",
      "registration": "19-11-2007",
      "state": "UT",
      "tdpId": 40
    },
    {
      "avgAmount": "163.42",
      "birth": "12-01-1980",
      "city": "Helena",
      "firstname": "Thomas",
      "id": "41",
      "lastname": "Eisenhower",
      "nbCommands": "33",
      "registration": "01-12-2007",
      "state": "KY",
      "tdpId": 41
    },
    {
      "avgAmount": "439.8",
      "birth": "06-08-1956",
      "city": "Charleston",
      "firstname": "Bill",
      "id": "42",
      "lastname": "Lincoln",
      "nbCommands": "46",
      "registration": "17-10-2007",
      "state": "WA",
      "tdpId": 42
    },
    {
      "avgAmount": "538.53",
      "birth": "25-12-1956",
      "city": "Dover",
      "firstname": "George",
      "id": "43",
      "lastname": "Tyler",
      "nbCommands": "68",
      "registration": "20-04-2008",
      "state": "TN",
      "tdpId": 43
    },
    {
      "avgAmount": "642.95",
      "birth": "25-07-1975",
      "city": "Montpelier",
      "firstname": "Franklin",
      "id": "44",
      "lastname": "Grant",
      "nbCommands": "52",
      "registration": "05-04-2007",
      "state": "LA",
      "tdpId": 44
    },
    {
      "avgAmount": "374.1",
      "birth": "18-12-1984",
      "city": "Lincoln",
      "firstname": "Martin",
      "id": "45",
      "lastname": "Monroe",
      "nbCommands": "10",
      "registration": "23-03-2007",
      "state": "MA",
      "tdpId": 45
    },
    {
      "avgAmount": "493.67",
      "birth": "15-05-1949",
      "city": "Salem",
      "firstname": "Richard",
      "id": "46",
      "lastname": "Madison",
      "nbCommands": "78",
      "registration": "17-05-2008",
      "state": "WY",
      "tdpId": 46
    },
    {
      "avgAmount": "616.05",
      "birth": "11-09-1973",
      "city": "Tallahassee",
      "firstname": "Dwight",
      "id": "47",
      "lastname": "Grant",
      "nbCommands": "54",
      "registration": "19-03-2007",
      "state": "AZ",
      "tdpId": 47
    },
    {
      "avgAmount": "498.38",
      "birth": "06-11-1980",
      "city": "Concord",
      "firstname": "Calvin",
      "id": "48",
      "lastname": "Roosevelt",
      "nbCommands": "70",
      "registration": "11-09-2008",
      "state": "MS",
      "tdpId": 48
    },
    {
      "avgAmount": "23.49",
      "birth": "20-08-1948",
      "city": "Boston",
      "firstname": "Franklin",
      "id": "49",
      "lastname": "Eisenhower",
      "nbCommands": "36",
      "registration": "26-08-2008",
      "state": "SC",
      "tdpId": 49
    },
    {
      "avgAmount": "741.53",
      "birth": "16-01-1951",
      "city": "Denver",
      "firstname": "Jimmy",
      "id": "50",
      "lastname": "Jefferson",
      "nbCommands": "31",
      "registration": "10-11-2008",
      "state": "IL",
      "tdpId": 50
    }
  ]
}